frontend/packages/dev-console/integration-tests/features/getting-started/getting_started _to_dev_perspective.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Create Sample Application
```

Comment:
Feature and scenario description doesn't match
In this feature, there is only one scenario, so I don't think Background is required
In general, Background is used, when multiple scenarios has same given statement, so instead of writing it multiple times, at one place given statement is available

Reply:
If the feature name is Create Sample Application
I recommend to add one more scenario to create an application, If not please add login with kubeadmin as well and explain the outcome in the scenario